core/domain/feedback_services.py

```diff
@@ -366,6 +366,34 @@ def _get_thread_from_model(thread_model):
         thread_model.created_on, thread_model.last_updated)
 
 
+def get_messages_for_exp_id(exp_id, limit=None):
```

Comment:
If the dashboard mentions "New Feedback", then as a user I would expect that the feedback messages be latest (irrespective of the thread they belong to). Thus, I think that might not be a very good option.